inference/app/inference.py

```diff
@@ -34,6 +41,32 @@ def allowed_filesize(filesize):
         return False
 
 
+class Models(object):
+    """Models"""
+
+    def __init__(self, model_id):
+        self.model_id = model_id
+        self.model_type = ['docker', 'binary', 's3']
+        self.current_data_state = ''
+        self.model_state = ''
+
+    def execute(self, current_data_state, model_state):
+        pass
+
+    def get_id(self):
+        return self.model_id
+
+
+class Registry(object):
```

Comment:
move this file to `/model_registry/models.py` 


The abstract base class should be like:
```
class Registry():
    def add_model():
    def remove_mode():
    def get_model():
```

You'd have a docker registry in `/model_registry/docker.py`
```
class DockerRegistry(Registry):
```
where you'd implement the methods. In a separate pull request when we're implementing the model registry. For this one just have the abstract classes and mock them in your tests.